test/verify/check-networking

```diff
@@ -120,7 +120,7 @@ class TestNetworking(MachineCase):
 
         # Configure a manual IP address
         #
-        b.click("tr:contains('IPv4') button")
+        b.click("tr:contains('IPv4') a", True)
```

Comment:
I had to use 'force' here because phantomjs doesn't consider the link to be visible.  Does anyone know what's going on?